13-tool-usage-cycle-tracking/README.md

```diff
@@ -377,28 +456,29 @@ This is an alert event (due to the presence of the `alert` property), which will
 
 ```json
 { "req": "note.add", "file":"power.qo", "sync": true, "body": {
-  "current":2.34, "frequency":60, "power":56.67, "voltage":230.12, "alert":"overcurrent,power", "instance":2, "active":true
+  "current":2.34, "frequency":60, "power":56.67, "voltage":230.12, "alert":"overcurrent,power", "instance":2, "active":true, "vibration":"normal", "vibration_raw":123
 }}
 ```
 
 This will send a SMS that looks like this:
 
-> Power alert from machine-12 equipment powered: yes: overcurrent,power. 120V, 25A, 3000W.
+> Power alert from <serial-number> tool active: yes: overcurrent,power. 120V, 25A, 3000W. vib.: normal,123.
```

Comment:
GitHub actually strips `<serial-number>` so it doesn’t appear in the rendered Markdown.

![Screenshot 2023-03-07 at 2 24 14 PM](https://user-images.githubusercontent.com/544280/223530032-b6e0d83e-27b7-4ee9-9a01-440ef26bb45a.png)